A3.md

```diff
@@ -87,7 +89,6 @@ The tracer files with expose several calls which with the C-core code can intera
 /* Initializes the tracing object with gpr_malloc. The caller has
    ownership over the returned tracing object */
 grpc_channel_tracer* grpc_channel_tracer_init_tracer();
```

Comment:
Should probably call this grpc_channel_tracer_create(), since it creates and returns a new object (as opposed to initializing a previously-allocated object).